### ISSUE TITLE
small issue but critical to make select query

### ISSUE THREAD

**Issue by libliboom** [jaredrummler/AndroidDeviceNames#39]
@jaredrummler There is issue on condition.

```
    else if (TextUtils.isEmpty(model)) {
      selection = COLUMN_MODEL + " LIKE ?";
      selectionArgs = new String[] { model };
    }
```
I think the condition will be edited like below
```
    else if (!TextUtils.isEmpty(model)) {
      selection = COLUMN_MODEL + " LIKE ?";
      selectionArgs = new String[] { model };
    }
```
I found this when I tried to fetch data with `Build.MODEL` first.

**Comment by rexi1r** [comment]
yes, that's right and it has one more problem. the SQL query to search device uses " or "in where condition but the cause of we have some devices with the wired model name we should use " and ".

for example my device information is : 
codename = "h1"
model = "LG-H860"

as you can see we have multiple devices with a combination of codename and models of mine.
![Screenshot from 2020-09-29 09-53-30](https://user-images.githubusercontent.com/4881902/94521369-ca47e480-023a-11eb-9992-e97523d04d3a.png)

and in library's, we have the same but as you can see with ' or ' condition additional result appears:
![Screenshot from 2020-09-29 09-54-29](https://user-images.githubusercontent.com/4881902/94521476-011dfa80-023b-11eb-96d1-7468fcb6dcf2.png)

and with " and " condition I can get the correct answer.
